package.json

```diff
@@ -126,6 +126,7 @@
     "@breejs/later": "4.0.2",
     "@iarna/toml": "2.2.5",
     "@renovate/pep440": "1.0.0",
+    "@renovatebot/parser-utils": "0.0.6",
```

Comment:
I'd like this to be `1.0.0` before we put this live in production

Reply:
@zharinov this should be bumped to V1.0.0 fIrst.